README.md

```diff
@@ -1,3 +1,12 @@
+Front-end redesign
+![screenshot](https://mediabuidlguidl.vercel.app/thumbnail.png)
+
+Font credits:
```

Comment:
Let's just use the one that Dhia provided.

Reply:
![47d2db85-1afd-48a5-9224-5e99662fc870](https://github.com/user-attachments/assets/f3a69877-848b-4488-99d6-8ff82d495370)

Let me know if you need a different format, I can ask him

Reply:
And if we need to give credit, we'll do it at the bottom of the readme with some text.

Doing two big screenshots at the top of the README is too much :D